less/api.less

```diff
@@ -1,54 +1,46 @@
-.monospace {
```

Comment:
This file is a bit of a mess. Here's how BEM works:

* one block (in this case, perhaps `api`)
* one `__element` (except at the root)
* optionally one `--modifier`

So we don't want things like `.api__description__header`. That's two elements.

In my opinion, this file was already BEM-ed; the fact that each block wasn't prepended with `api` wasn't a problem for me. If you'd rather have all these blocks namespaced, then let's prepend `api-` to each one and otherwise leave them as they were.

(The `.monospace` and `:root` selectors at the top have, I imagine, been moved elsewhere, so I'm not asking for those to be restored.)

Reply:
I see what you're saying. I certainly may have gone overboard with renaming things, getting excited about some kind of standardization. But I can certainly go back through fixing up the naming scheme.

Since I still feel like we want to prepend them with `api` I'll take your suggestion of `api-` and re-evaluating how I changed them in the first place. Thanks for the time you've taken for a review!